src/structs/slash.rs

```diff
@@ -18,7 +18,7 @@ pub enum ApplicationCommandOrAutocompleteInteraction<'a> {
 
 impl<'a> ApplicationCommandOrAutocompleteInteraction<'a> {
```

Comment:
Can be renamed to CommandOrAutocompleteInteraction

Reply:
Are you implying you will merge ApplicationCommandInteraction and AutocompleteInteraction in upstream serenity soon? Interesting